### ISSUE TITLE
t or Split is undefined 8.17.4 but nothing when rollback 

### ISSUE THREAD

**Issue by GoatFreezy** [kazupon/vue-i18n#868]
Hello,
We have our CI which updated vuei18n to the lastest version (8.17.4) at our last deployement.
unfortunatly this cause our whole website to crash with 2 different errors :
"t is undefined"
"TypeError: Cannot read property 'split' of undefined"

We stuggled on detecting it because with the browser cache the website was correctly loading, so all developpers didnt had the error.

I was able to make our website functionnal after rolling back to the version we used to have : 8.12.4.

Our major probleme is that with the browser cache in local we dont have any error, it only happends on servers.

**Comment by pegiadise** [comment]
Same thing here. It happens when `vue-i18n` is set to a locale that is not listed in browser locales. Reverting to `v8.16.0` which was the last one working for me.

**Comment by pegiadise** [comment]
Happens [here](https://github.com/kazupon/vue-i18n/blob/078d77489f8124a91f747e693709779b939a44c1/src/index.js#L480) and fails with `TypeError: Cannot read property 'split' of undefined`. Sentry shows that the error is thrown on render. I attach the respective (unfortunately minified) error snippet below.

```
{snip} U.prototype._appendLocaleToChain=function(t,e,n){var i,r=e.split("-");do{var a=r.join("-");i=this._appendItemToChain(t,a,n),r.splice(-1,1)}w {snip}
```

Hope it helps a bit.

**Comment by kazupon** [comment]
Fixed.
I've just released v8.17.5
please check it!
https://github.com/kazupon/vue-i18n/releases/tag/v8.17.5

**Comment by pegiadise** [comment]
v8.17.5 resolved the issue indeed.
